src/ontology/mappings/d3fend-cco.ttl

```diff
@@ -59,6 +66,76 @@ d3f:Agent owl:equivalentClass cco:Agent . # Tentative, see note below.
   # associated with agent-based model of thinking.)
 
 
+d3f:Organization rdfs:subClassOf cco:Organization.
+
+# d3fend only allows persons to be members of organizations, while CCO also allows 
+# other organizations to be part of an orgaization. Thus the CCO class is slightly broader.
+
+d3f:Person owl:equivalentClass cco:Person.
+
+# d3fend seems to only refer to human beings with "person", just as CCO does.
+
+d3f:PhysicalObject rdfs:subClassOf <http://purl.obolibrary.org/obo/BFO_0000030>.
+
+# BFO 0000030 is "object". The two classes might be equivalent, but there may also be
+# other classes in d3fend which are not subclasses of PhysicalObject that fall under
+# BFO:0000030
+
+d3f:Technique rdfs:subClassOf cco:DirectiveInformationContentEntity.
```

Comment:
This will require some discussion and has been a big pain point for us to reconcile ontology with industry colloquialisms.